### PR TITLE
Update TLDRLegal.com.xml

### PR DESCRIPTION
**Pull request by koops76**
#11114

### CODE REVIEW

File: src/chrome/content/rules/TLDRLegal.com.xml [hunk 1]
```diff
@@ -1,30 +1,9 @@
...
-	These altnames don't exist:
-
-		- dev.tldrlegal.com
-
--->
-<ruleset name="TLDRLegal.com" default_off='failed ruleset test'>
-
+<ruleset name="TLDRLegal.com">
 	<target host="tldrlegal.com" />
 	<target host="www.tldrlegal.com" />
```
Comment (cschanaj) [inline src/chrome/content/rules/TLDRLegal.com.xml]: please document the `community` and `www` in the comment.

Comment (koops76) [inline src/chrome/content/rules/TLDRLegal.com.xml]: Do these subdomains still exist?

Comment (koops76) [inline src/chrome/content/rules/TLDRLegal.com.xml]: Moved the comment to the top.

Comment (koops76) [inline src/chrome/content/rules/TLDRLegal.com.xml]: Added the `community` subdomain.

**Pull request merged**